Cargo.toml

```
@@ -74,9 +74,10 @@ features = ["serde"]
 version = "1.0.0"
 
 [dev-dependencies]
-assert_cli = "0.6.3"
+predicates = "2.0"
+assert_cmd = "2.0.1"
+assert_fs= "1.0.5"
```

you didn't use `cargo add` to add this, huh?
```suggestion
assert_fs = "1.0.5"
```